.circleci/config.yml

```diff
@@ -1,29 +1,49 @@
 version: 2.1
 
+orbs:
+  browser-tools: circleci/browser-tools@1.4.8
+
+executors:
+  node:
+    docker:
+      - image: cimg/node:18.18
+  node-browsers:
+    docker:
+      - image: cimg/node:18.18-browsers
+
 jobs:
   execute-npm:
     parameters:
+      executor:
+        type: executor
+        default: node
       npm-run-command:
         type: string
-    docker:
-      - image: cimg/node:18.18
+      setup:
+        type: steps
+        default: []
+    executor: << parameters.executor >>
     steps:
       - checkout
+      - steps: << parameters.setup >>
       - run: npm ci
       - run: npm run << parameters.npm-run-command >>
 
 workflows:
   main:
     jobs:
-      - execute-npm:
-          name: npm run build
-          npm-run-command: build
```

Comment:
Please keep `build` as a dedicated job. Otherwise if "lighthouse" fails it is not clear, if this is because of a broken build or if some lighthouse score was not achieved.